src/space_usage.rs

```diff
@@ -0,0 +1,110 @@
+//! Module to collect and display Disk Space Usage of a Profile.
```

Comment:
For naming, I have checked and other messengers call this "Storage usage", not "Space usage":
- [Telegram](https://telegram.org/blog/hidden-media-zero-storage-profile-pics/) has a dedicated "Storage usage" screen
- Signal has a "Storage usage" screen accessible via "Data and storage" -> "Manage storage"
- [WhatsApp](https://faq.whatsapp.com/5503646096388294/)
- Android itself has "Storage and cache" and "Storage" screens for each app

So I think it is better to rename the module into `storage_usage` or `storage` and use term in the APIs and for the report title. Otherwise we will have yet another case of naming mismatch between the API and the UI (similar to account <-> profile, ephemeral messages <-> disappearing messages).